src/app/infra/views/sensors_records_views/last_sensors_record_page_view.py

```diff
@@ -10,7 +12,7 @@ def last_sensors_record_page_view():
 
     if last_data:
```

Comment:
If there is no the last record, then the last record should be:

```python
{
"temperature": 0,
"water_volume": 0,
"soil_humidity": 0,
"ambient_humidity": 0,
}

```

Reply:
What about the date